config/install/core.entity_view_display.media.video.default.yml

```diff
@@ -0,0 +1,48 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.media.video.field_media_video_embed_field
```

Comment:
Let's be consistent here. We named the Image field "field_image". So this should be just "field_video".